### PR TITLE
Ui enhancement

### PR DESCRIPTION
**Pull request by kyoblaze**
Rearranging all the layout to meet better user's interface design.

### CODE REVIEW

File: src/redux/modules/favourite.js [hunk 1]
```diff
@@ -0,0 +1,191 @@
+// @flow
+type Place = {
+  latitude: number;
```
Comment (vshy108) [inline src/redux/modules/favourite.js]: the key should end with , instead of ; for flow type definition

File: src/containers/Home/FavList.js [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+
+  renderBACKButton() {
+    return (
+      <TouchableHighlight
+        onPress={() => this.modeSwitch()}
+        style={{
+          width: '15%', height: '50%', marginRight: 20, marginTop: 15,
+        }}
+      >
+        <Text style={{ textAlign: 'center', fontSize: 15 }}>BACK</Text>
```
Comment (vshy108) [inline src/containers/Home/FavList.js]: style should put in its own style file

File: src/containers/Home/FavList.js [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+          style={{
+            flex: 1, alignItems: 'flex-end', justifyContent: 'center',
+          }}
+        />
+        {this.props.favMode ? this.renderBACKButton() : null}
+      </View>
+    );
+  }
+
+  renderBACKButton() {
```
Comment (vshy108) [inline src/containers/Home/FavList.js]: follow camel case naming style

File: src/containers/Home/FavList.js [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+import React from 'react';
+import {
+  View,
+  ScrollView,
+  Modal,
+  Text,
+  TouchableHighlight,
+} from 'react-native';
+import { connect } from 'react-redux';
+import styles from './style';
```
Comment (vshy108) [inline src/containers/Home/FavList.js]: If FavList included in its own folder, then no need share style file with Home

File: src/containers/Home/FavList.js (hunk 2 above)
Comment (kyoblaze) [reply src/containers/Home/FavList.js]: if some like only one phrase or two phrase also need to put into style file meh?

**Pull request merged**